### ISSUE TITLE
Option to surpress output file listing

### ISSUE THREAD

**Issue by gijzelaerr** [common-workflow-language/cwltool#583]
hi!

I'm using cwltool and i'm missing a feature to supress the listing of output products. When a pipeline has a lot of output products the console gets filled up with data which is quite pointless. I tried `--quiet` but that doesn't seem to do much.

thanks!

**Comment by mr-c** [comment]
Hey @gijzelaerr . Are you wanting to suppress the CWL output object JSON, or the other output?

**Comment by gijzelaerr** [comment]
hi @mr-c! the JSON :)

**Comment by mr-c** [comment]
@gijzelaerr `cwltool --quiet mydescription.cwl > /dev/null` :-)

**Comment by gijzelaerr** [comment]
so console output goes to stderr?

**Comment by mr-c** [comment]
STDERR and STDOUT from the underlying tool are not redirected unless specified in the CommandLineTool description. Would you like them to be suppressed when `--quiet`?

**Comment by gijzelaerr** [comment]
No, I would like to see the output of the encapsulated program, without the verbose JSON.

**Comment by gijzelaerr** [comment]
I think the JSON output in most cases is not very useful, it is probably much more useful to have a brief overview of output products and types.
